contracts/income/MarketMaker.sol

```diff
@@ -36,15 +36,15 @@ contract MarketMaker {
   }
 
   function decimals() public view returns (uint8) {
-    return dai.decimals();    
+    return token.decimals();    
```

Comment:
`balanceOf` returns the balance in DAI. Shouldn't we have decimals for DAI here, so that amount is rendered right?